README.md

```diff
@@ -13,21 +12,45 @@ Teams spend significant time building parts of the design system (colors, variab
 [![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
 ![license](https://img.shields.io/github/license/mashape/apistatus.svg)
 
+:star: We appreciate your star, it helps!
+
+## Key features & benefits
+### Unopionated
+These days, there are a million and one great ways to build your next Web site or application. So, we have made being unopinionated the center of every decision made in FAST-DNA. 
+
+This principle of being unopinionated manifests in several important ways including:
+* A flat component architecture that lets you compose what you need without struggling with rigid patterns and complex objects.
+* Separating base components and style that supports multiple frameworks without re-writing or duplicating styles. This separation also lets you build your own styles, with your style technology of choice, without having to re-build or duplicate the base components.
+* Framework agnostic tooling that lets you use our Development tools with any view framework.
+* The ability to replace almost any FAST-DNA package with your package of choice. Get started with our animation or layout package and add more as you need them. Alternatively, use our suite of packages to build your next project from the ground up; it's your call.
+
+### UI development and style guide tools
+When developing components and views, excellent development tooling can make all the difference. FAST-DNA offers development and style guide tools that are with FAST components, components from other frameworks or your components. 
+
+Try out component properties with an auto-generated props UI, get a live preview of the code based on any property’s configuration, preview RTL and themes, and preview component compositing with a transparency grid.
+
+Also, we built FAST development tools from re-usable packages so, if you have special needs, you can build your tools from the same shared libraries.
+
+### Bring your design system
+Widely available design systems from companies like Microsoft (Fluent), Google (Material), or Salesforce (Lightning) are useful when it is essential to align with a platform or take advantage of a polished system at low cost, but many companies have their design system of their own, and some may have multiple design systems or variations.
+
+Because FAST-DNA has abstracted based components from their style, you get a head start on your design system by building on top of tried and true base components and style libraries like our offerings for color, animation, and elevation.
```

Comment:
Because FAST-DNA has abstracted "base" components

Reply:
I think it's mentioned in the unopinionated section, should we repeat?

Reply:
This is saying something different, the one above is saying that you can use different base component and styling layers and mix and match them. The second is talking about how we can accelerate bringing your own design system. Going to leave for now but will be tuning this messaging a lot over the coming weeks. I will keep an eye on it.